app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor has no vaccine."""
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when vaccine has been expired."""
+    pass

```

Comment:
Consider adding a descriptive error message to the `OutdatedVaccineError` exception. For example: `class OutdatedVaccineError(VaccineError): def __init__(self, message='The vaccine is outdated'): super().__init__(message)`.